app/serializers/search_and_compare/course_serializer.rb

```diff
@@ -23,45 +23,98 @@ class CourseSerializer < ActiveModel::Serializer
     # using server time not utc, so it's local time?
     attribute(:StartDate)                             { object.start_date.utc.strftime('%Y-%m-%dT%H:%M:%S') }
 
-
     # Salary_nested_default_value_Mapping
-    attribute(:Salary)                                { default_salary }
-
-    def default_salary
-      {
-        Minimum: nil,
-        Maximum: nil,
-      }
-    end
+    attribute(:Salary)                                { default_salary_value }
 
     # Subjects_related_Mapping
     attribute(:IsSen)                                 { object.is_send? }
-    attribute(:CourseSubjects)                        { get_subjects }
+    attribute(:CourseSubjects)                        { course_subjects }
+
+
+    # Campuses_related_Mapping
+    attribute(:Campuses)                              { campuses }
+    # using server time not utc, so it's local time?
+    attribute(:ApplicationsAcceptedFrom)              { object.applications_open_from.to_date.strftime('%Y-%m-%dT%H:%M:%S') }
```

Comment:
Not clear on what's going on here ... `applications_open_from` is a date, does search-and-compare store it as a time?

Reply:
input
`'2018-10-09T00:00:00'`

expected output 
`'2018-10-09T00:00:00'`

actual output
`'2018-10-09T00:00:00Z'`

Notice the `Z` for the time zone

Reply:
I just checked and it is a `timestamp` (no tz :( ).